docs/REPRESENTER_NORMALIZATIONS.md

```diff
@@ -0,0 +1,85 @@
+# Representer normalizations
+
+The [JavaScript representer][github-javascript-representer] applies the following normalizations:
+
+## Remove comments
+
+All comments are removed.
+The following examples are equivalent:
+
+### Example with comments
+
+```javascript
+/**
+ * Returns the string literal 'hi there' just to say hello
+ * @return [String]
+ */
+function hello(/* nothing*/) {
+  // just return it
+  return 'hi there';
+}
+```
+
+### Example without comments
+
+```javascript
+function hello() {
+  return 'hi there';
+}
+```
+
+## Normalize whitespace
+
+When the code is represented, it's _order_ is significant, but its physical location is not.
+This means that whitespace is normalized.
+The following examples are equivalent:
+
+### Example with two-space indentation
+
+```javascript
+function hello() {
+  return 'hello world';
+}
+```
+
+### Example with four-space indentation
+
+```javascript
+function hello() {
+  return 'hello world';
```

Comment:
This should have 4 spaces right?

Reply:
It's currently formatting with 2 spaces, but it's configurable

Reply:
Oh. Hahaha. Yes. Prettier fixed it. Will ignore pretti r!